File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String s = "";
        String c = "";
        for (String i: word1){
            s += i;
       }
        for (String d: word2){
            c += d;
        }
        return s.equals(c);
    }
    
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.cpp
#include<string.h>

class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        std::string s = "";
        std::string c = "";
        for (string i : word1)
        {
            s += i;
        }
        for (string d : word2)
        {
            c += d;
        }
        int res = s.compare(c);
        return res == 0;

    }
};